frontend/src/metabase/palette/components/PaletteResultsList.tsx

```diff
@@ -120,7 +120,7 @@ export const PaletteResultList: React.FC<PaletteResultListProps> = props => {
       if (item.command) {
         item.command.perform(item);
         query.toggle();
-      } else {
+      } else if (!item.extra?.href) {
```

Comment:
With this change, we don't have to include a no-op `perform` function when adding an external link to the command palette results.

Reply:
Nice!